google/ggrc-core
# [Matrix view] GGRC-8745 Set up logic related to validation

# Comment
**Should be merged to "matrix-view".**

# Dependencies

This PR is `on hold` until the dependencies are merged:
- [x] #10784 

# Issue description

Set up logic related to validation

# Steps to test the changes

1. Open My Assessments/ Assessment tab on Audit.
2. Click "Bulk Completion" button.
3. Table should be shown.
    - validation should work as on Assessment Info Page
    - row should be colored blue when each attribute in this row is valid and tick should be shown near assessment title
    - 'Save answers' button should be enabled when there is at least one not saved change in the table
    - 'Complete(n)' button should be enabled when all attributes in at least one row are valid, n - number of valid rows

# Solution description

Add Required Info modal and update validation logic.

# Sanity checklist




# PR Review checklist
